sycl/test-e2e/Basic/fpga_tests/fpga_io_pipes.cpp

```diff
@@ -1,6 +1,10 @@
+// The test is disabled, since no support added in FPGA backend yet. Check
+// implementation correctness from CXX and SYCL languages perspective.
+// UNSUPPORTED: true
```

Comment:
NOTE: As of now, this test has some value - it validates the code compiles. Your change reduces the testing scope.

I would just remove line 3 (i.e. `// RUNx: %{run} %t.out`) and keep the TODO comment about adding execution.

This test should be moved from e2e tests to `check-sycl` suite. We should also check with FPGA team if there are plans to enable execution part, if so, we need to file a tracker.

Reply:
> NOTE: As of now, this test has some value - it validates the code compiles. Your change reduces the testing scope.

Right, thanks, didn't think about that. I will keep the first `RUN` and file an issue to track the further progress.